### PR TITLE
fix(dev): release notification scripts works with latest ts-node

### PR DESCRIPTION
**Pull request by geigerzaehler**
To make the `set-latest-release` script work with the latest version of `ts-node` we add `tsconfig.scripts.json` with `module: commonjs`. We also change the URL for the info file to `releases.radicle.xyz/latest.json`.

### CODE REVIEW

**Comment by rudolfs** [comment]
```
$ npx ts-node ./scripts/set-latest-release.ts
Copying file://radicle-dev-set-latest-releaseFTO2Ur/latest.json [Content-Type=application/json]...
/ [1 files][  106.0 B/  106.0 B]
Operation completed over 1 objects/106.0 B.
TypeError: fs.promises.rm is not a function
    at withTempDir (/Users/rudolfs/work/radicle-upstream-2/scripts/set-latest-release.ts:77:23)
    at main (/Users/rudolfs/work/radicle-upstream-2/scripts/set-latest-release.ts:24:3)
```

**Review by xla — APPROVED** [review]
👆 ↖️ 🎅 🏞

**Comment by rudolfs** [comment]
It also seems to leave files in the repo:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	radicle-dev-set-latest-releaseFTO2Ur/
```

**Comment by geigerzaehler** [comment]
@rudolfs: Seems like your using an old version of node. [`fs.promises.rm`](https://nodejs.org/dist/latest-v14.x/docs/api/fs.html#fs_fspromises_rm_path_options) was introduced in v14.14. I will update `.node-version` accordingly.

**Comment by rudolfs** [comment]
```
work/radicle-upstream-2 » node --version
v14.5.0
```

**Comment by geigerzaehler** [comment]
@rudolfs could you try if the fix I pushed works on macOS?

**Comment by rudolfs** [comment]
```
$ node --version
v14.15.4
$ ./scripts/set-latest-release.ts
⨯ Unable to compile TypeScript:
scripts/set-latest-release.ts:14:19 - error TS2307: Cannot find module 'node-fetch' or its corresponding type declarations.

14 import fetch from "node-fetch";
                     ~~~~~~~~~~~~
```

**Comment by rudolfs** [comment]
Also I just realised that `asdf` doesn't use `.node-version` but `.tool-versions` and should contain:
```
nodejs 14.15.4
```

We can remove the `.node-version` symlink.

**Comment by rudolfs** [comment]
The script seems to work:
```
work/radicle-upstream-2 » ./scripts/set-latest-release.ts


Updates are available for some Cloud SDK components.  To install them,
please run:
  $ gcloud components update

Copying file:///var/folders/3g/zdz1p3xs55s7s_f2qn8_g2r80000gn/T/radicle-dev-set-latest-release-s7yZ1r/latest.json [Content-Type=application/json]...
/ [1 files][  106.0 B/  106.0 B]
Operation completed over 1 objects/106.0 B.
```

**Review by rudolfs — DISMISSED** [review]

**Review by rudolfs — APPROVED** [review]

File: scripts/set-latest-release.ts [hunk 1]
```diff
@@ -70,16 +72,12 @@ async function main() {
 async function withTempDir(
   cb: (tempDir: string) => Promise<void>
 ): Promise<void> {
-  const tempDir = await fs.promises.mkdtemp("radicle-dev-set-latest-release");
+  const tempDir = await fs.promises.mkdtemp(
+    path.join(os.tmpdir(), "radicle-dev-set-latest-release-")
```
Comment (NunoAlexandre) [inline scripts/set-latest-release.ts:76]:
```suggestion
    path.join(os.tmpdir(), "radicle-dev-set-latest-release")
```

is that `-` intentional?

**Review by NunoAlexandre — APPROVED** [review]
Cool! Just left a question, looks good otherwise :100:

File: scripts/set-latest-release.ts (hunk 1 above)
Comment (geigerzaehler) [reply scripts/set-latest-release.ts:76]: Yes. The base string will be suffixed with some random string. To make it look nicer I added the dash.

**Comment by geigerzaehler** [comment]
Merged in 5dcaf7caed9a5f6076686212ca2ef9a31e021073

**Pull request closed without merging**